solutions/standard/catalogValidationValues.json.template

```diff
@@ -0,0 +1,5 @@
+{
+  "ibmcloud_api_key": $VALIDATION_APIKEY,
+  "existing_secrets_manager_crn": "crn:v1:bluemix:public:secrets-manager:us-south:a/abac0df06b644a9cabc6e44f55b3880e:79c6d411-c18f-4670-b009-b0044a238667::",
```

Comment:
You can use `$SM_CRN` as the value here and pipeline will replace it with the correct CRN